charts/shoot-core/components/charts/vpn-shoot/templates/secret-tlsauth.yaml

```diff
@@ -1,12 +1,9 @@
+# TODO(shafeeqes): Delete this Helm chart in a future version.
 ---
 apiVersion: v1
 kind: Secret
 metadata:
   name: {{ include "vpn-shoot.tls-auth.name" . }}
```

Comment:
I think you have removed _helpers.tpl so the name can't be included here in this way? I think to make files consistent leave the name part in the _helpers.tpl

Reply:
Good catch. will do.